wrapper/pycc/src/qcc_db/ccObject.cpp

```diff
@@ -134,6 +142,15 @@ void define_ccObject(py::module &m)
         .def("swapChildren", &ccHObject::swapChildren, "firstChildIndex"_a, "secondChildIndex"_a)
         .def("getChildIndex", &ccHObject::getChildIndex, "child"_a)
         .def("getIndex", &ccHObject::getIndex)
+        .def(
+            "addChild",
+            [](py::object &self, py::object &child, ccHObject::DEPENDENCY_FLAGS dependencyFlags, int insertIndex) {
+                self.cast<ccHObject *>()->addChild(child.cast<ccHObject *>(), dependencyFlags, insertIndex);
+            },
+            py::arg("child"),
+            py::arg("dependencyFlags") = ccHObject::DEPENDENCY_FLAGS::DP_PARENT_OF_OTHER,
+            py::arg("insertIndex") = -1
```

Comment:
To be consistent with the rest of the code I would use the `_a` suffix eg `"insertIndex"_a=-1`